docs/portal.properties-Reference.md

```diff
@@ -61,52 +75,59 @@ authorization=true
 ```
 oncoprint.defaultview=sample
 ```
 
-## Custom annotation of driver and passenger mutations
-cBioPortal allows for two different formats of custom annotation of driver and passenger mutations in the MAF files (see [file format documentation](File-Formats.md#extending-the-maf-format)). To enable the view of those annotations (just one or all both), you must specify a name for the label(s) that you want to enable (those labels appear in the oncoprint "Mutation color" menu):
-```
-oncoprint.custom_driver_annotation.binary.menu_label=Custom driver annotation
-oncoprint.custom_driver_annotation.tiers.menu_label=Custom tiers
-```
+# Custom annotation of driver and passenger mutations
+cBioPortal supports 2 formats to add custom annotations for driver and passenger mutations. 
+1. **cbp_driver**: This will define whether a mutation is a driver or not.
+2. **cbp_driver_tiers**: This can be used to define multiple classes of driver mutations. 
 
-By default, when at least one of the custom annotations is enabled in portal.properties and the required columns are present in MAF files, custom annotations are automatically selected, instead of OncoKB and Hotspots. If you wish to change this default behavior and have OncoKB and Hotspots as the default selected annotations, set this parameter to false:
+These data formats are described in the [cBioPortal MAF specifications](File-Formats.md#extending-the-maf-format). 
+
+#### Enabling custom annotations in the OncoPrint
+To enable functionality for one or both types of custom annotations, enter values for the following properties. These labels will appear in the OncoPrint's "Mutation color" menu.
 ```
- oncoprint.custom_driver_annotation.default=false
+oncoprint.custom_driver_annotation.binary.menu_label=Custom driver annotation
+oncoprint.custom_driver_annotation.tiers.menu_label=Custom driver tiers
 ```
 
-By default, when at least one of the custom annotations is enabled in portal.properties and the required columns are present in MAF files, custom annotations are automatically selected. You can change this behaviour and set the following properties to "false" to automatically unselect one or both custom driver/passenger mutations (by default, those properties are set to true):
+#### Automatic selection of custom annotations
+By default, when at least one of these custom annotations is enabled in `portal.properties` and the required columns are present in MAF files, custom annotations are automatically selected instead of OncoKB and Hotspots. To disable this default behavior and have OncoKB and Hotspots as the default selected annotations, set the following properties to `false`.
 ```
-oncoprint.custom_driver_annotation.default=false
-oncoprint.custom_driver_tiers_annotation.default=false
+oncoprint.custom_driver_annotation.default=true
+oncoprint.custom_driver_tiers_annotation.default=true
 ```
```

Comment:
@sandertan this should prolly stay false right?

Reply:
By setting `true`, @oplantalech implemented in a way that custom driver annotations are only enabled when data is present, so it should not influence instances without this data.

We could make it `false` to make it an conscious decision when you want to enable this functionality.

Reply:
The documentation should just list the options. I like how civic option is documented above:

```
CIViC integration can be turned on or off with the following property (default: true):
show.civic=true|false
```

Reply:
@pieterlukasse Done. Also made docu "Automatic selection of custom annotations" even simpler.